### PR TITLE
Docs: Removed condescending language

### LINKED ISSUE #722: More inclusive language in docs
:wave: I noticed that there are a couple of instances of words like "simply" in your documentation. Unfortunately, words like this can make people feel frustrated and isolated while reading (myself included, hence the issue) - especially if they are facing issues.

If it's alright with your team, I'd like to take a shot at updating the content to make it more inclusive by removing instances of "simply", "easy", "just", etc. I've been following you all since back in the Graphcool days, so I'm happy to help 😊  

Also if you're interested or want more information on this topic, I'd highly recommend watching Jim Fisher's [Don't Say Simply](https://www.youtube.com/watch?v=gsT2BBWBVmM) talk from Write the Docs Prague 2018 ✍️ 🤓 

Thank you for the work you do and please let me know if you need more details!

### PR DESCRIPTION
**Pull request by carolstran**
Closes #722  

As discussed in the issue, this PR attempts to remove as many instances of condescending or isolating language (ie. simple, easy, just, of course, etc) as possible in the documentation. While most of the changes are removing the word, some involved altering the text. 

Disclaimer: Feel free to revert or ask me to fix any parts that you believe should stay the way they are. If even a fraction of these changes can be merged then it will be a huge step towards making the documentation more inclusive!

### CODE REVIEW

File: docs/photon/codegen-and-node-setup.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ node_modules/@generated/photon
...
 
-Node.js libraries are typically installed as npm dependencies using `npm install`. The respective packages are then located inside the [`node_modules`](https://docs.npmjs.com/files/folders#node-modules) directory from where they can be easily imported into application code.
+Node.js libraries are typically installed as npm dependencies using `npm install`. The respective packages are then located inside the [`node_modules`](https://docs.npmjs.com/files/folders#node-modules) directory from where they can be imported into application code.
 
 Because Photon.js is a custom API for _your_ specific database setup, it can't follow that model. It needs to be generated locally instead of being installed from a central repository like npm. However, the mental model for Photon.js should still be that of an Node module.
 
 ## Why is Photon.js generated into `node_modules/@generated` by default?
 
-### Easy imports
+### One-line imports
```
Comment (pantharshit00) [inline docs/photon/codegen-and-node-setup.md]:
```suggestion
### Non-relative imports
```


One-line import isn’t the correct term here as all imports in JS/TS are one liner.

**Review by pantharshit00 — CHANGES_REQUESTED** [review]

File: docs/photon/codegen-and-node-setup.md (hunk 1 above)
Comment (janpio) [reply docs/photon/codegen-and-node-setup.md]: Is any qualifier for "imports" needed here? 
`Importing Photon.js` could work as well.

Comment (carolstran) [reply docs/photon/codegen-and-node-setup.md]: Totally agree!

File: docs/tutorial.md [hunk 2]
```diff
@@ -234,7 +234,7 @@ The project also contains a number of additional files required for a typical fo
 - `package.json`: Defines your project's Node.js dependencies.
 - `tsconfig.json`: Specifies your TypeScript configuration. Note that Photon.js currently requires the `esModuleInterop` property to be set to `true`.
 - `node_modules/@generated/photon`: Contains the generated Photon.js code. 
-- `script.ts`: Contains the actual "application code", which in this case is a simple script demonstrating some Photon.js API calls.
+- `script.ts`: Contains the actual "application code", which in this case is a script demonstrating some Photon.js API calls.
```
Comment (janpio) [inline docs/tutorial.md]: Maybe `basic` if we want to highlight that the script is not complicated?

Comment (carolstran) [reply docs/tutorial.md]: Your call. Let me know and I'll update with the other comments!

File: docs/photon/codegen-and-node-setup.md (hunk 1 above)
Comment (pantharshit00) [reply docs/photon/codegen-and-node-setup.md]: I also agree with @janpio 

```suggestion
### Importing Photon.js
```

File: docs/tutorial.md (hunk 2 above)
Comment (pantharshit00) [reply docs/tutorial.md]: Maybe `sample` can work here better

**Review by pantharshit00 — APPROVED** [review]

**Comment by nikolasburk** [comment]
Thanks so much @carolstran 🎉

**Pull request merged**